src/home/home-util.c

```diff
@@ -127,6 +127,16 @@ int bus_message_append_secret(sd_bus_message *m, UserRecord *secret) {
         return sd_bus_message_append(m, "s", formatted);
 }
 
+int bus_new_method_call(sd_bus *bus, sd_bus_message **m, const char *member) {
```

Comment:
hmm, this should be OK to do, but the function name really should make clear this is just about calls to homed, i.e. maybe call this home_method_call_new()?

Reply:
I'm going to close this PR because I think we're going to take a different approach as described in https://github.com/systemd/systemd/pull/15284#issuecomment-607637157